lib/sequelize.js

```diff
@@ -219,8 +219,19 @@ class Sequelize {
 
         if (options.dialectOptions)
           Object.assign(options.dialectOptions, urlParts.query);
-        else
+        else 
+        {
```

Comment:
This change introduced braces for `else` part, so it will make sense to add braces to preceding `if` statement as well.

And use `else {`

Reply:
Done